main.nf

```diff
@@ -11,17 +11,37 @@ include {
 } from "./subworkflows/encyclopedia"
 
 
+//
+// Used for email notifications
+//
+def email() {
+    def msg = TalusTemplate.email(workflow, params)
+    if (params.email) {
+        sendMail(
+            to: "$params.email",
```

Comment:
Not specific for this line of code, but is there a way we can document all the params? It would be nice to easily see what all the parameters are and what they are used for.

Reply:
I wish there was an easy way to do this. [nf-core](https://nf-co.re/) has adopted a JSON file format for document their parameters. Here's an example: https://github.com/nf-core/isoseq/blob/master/nextflow_schema.json

Such a file would at least let us build a documentation page automatically, but would be part of separate PR.